README.md

```diff
@@ -2,6 +2,25 @@
 
 # Distributed Linear Algebra from the Future
 
+DLA-Future is a distributed linear algebra library implemented using C++ proposal P2300 which provides:
+- An asynchronous C++ interface,
+- A synchronous C interface,
+- A synchronous ScaLAPACK-like C interface (ScaLAPACK drop-in with a subset of ScaLAPACK arguments. E.g. workspace arguments are not present),
+- A synchronous ScaLAPACK-like Fortran interface (See [DLA-Future-Fortran](https://github.com/eth-cscs/DLA-Future-Fortran)).
+
+Currently it provides the following algorithms :
+- Matrix Multiplication:
+  - Triangular (`P[S,D,C,Z]TRMM`)
+  - Hermitian (`P[C,Z]HEMM` and `P[S,D]SYMM`)
+- Triangular Solver (`P[S,D,C,Z]TRSM`)
+- Cholesky Decomposition
+- Symmetric / Hermitian eigenvalue solver (`P[C,Z]HEEVD` and `P[S,D]SYEVD`. Can replace any call to ScaLAPACK`P[C,Z]HEEV*` and `P[S,D]SYEV*`.)
+- Generalized Symmetric / Hermitian eigenvalue solver (`P[C,Z]HEGVD` and `P[S,D]SYGVD`.  Can replace any call to ScaLAPACK `P[C,Z]HEGV*` and `P[S,D]SYGV*`.)
```

Comment:
?

```suggestion
- Generalized Symmetric / Hermitian eigenvalue solver (`P[C,Z]HEGVD` and `P[S,D]SYGVD`. It can replace any call to ScaLAPACK `P[C,Z]HEGV*` and `P[S,D]SYGV*`).
```

Reply:
Outdated